### DISCUSSION TITLE
Automation Icon color changes based on "on/off" toggle

### DISCUSSION THREAD

**Discussion by jlr2000** [home-assistant/frontend#21072] [Automations / scripts / scenes]
In the Automations page it would be helpful visually if the icon of the automation changed color based on state; for example, if the automation is toggled "on" then the icon for the automation would be yellow, if toggled "off" then the icon would be in the default white color.  
This would give the user a quick way to see if an automation is enabled or not, without looking to the far right of the automation line (or having to scroll over when using a smaller screen such as a phone or tablet).  Customization of the colors would be great as well if possible.
![image](https://github.com/home-assistant/frontend/assets/26529911/0f8d5a54-843f-444a-b36c-c6c55d4ab1eb)

Thank you for considering!!

**Comment by ildar170975** [comment]
Probably, similarly ALL icons should be colored on "Entities" page dependently on a domain/state.
